src/Protector.php

```diff
@@ -881,6 +884,16 @@ public function isExecEnabled(): void
         }
     }
 
+    /**
+     * Stops Telescope from recording when telescope is used.
+     */
+    public function stopTelescopeRecordingIfEnabled(): void
+    {
+        if (class_exists(Telescope::class)) {
+            Telescope::stopRecording();
```

Comment:
You might want to use the FQN here instead:
```suggestion
        if (class_exists(\Laravel\Telescope\Telescope::class)) {
            \Laravel\Telescope\Telescope::stopRecording();
```

Reply:
Done.